src/ResourceManager/Compute/Commands.Compute/Commands.Compute.Netcore.csproj

```diff
@@ -23,23 +27,20 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
-    <PackageReference Include="Hyak.Common" Version="1.1.3" />
-    <PackageReference Include="Microsoft.Azure.Common" Version="2.1.4" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.Authentication" Version="2.2.12" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```

Comment:
We will need to repl;ace these with common code references fairly soon, but this needs to happen for the deskop version as well